app/javascript/components/RoomCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useCallback } from "react";
+import { useNavigate } from "react-router";
+import { Card, Container } from "react-bootstrap";
+import ButtonLink from "./stylings/buttons/ButtonLink";
+import { PersonSquare, Link45deg } from 'react-bootstrap-icons';
+
+export default function RoomCard(props) {
+  const  {id, name} = props
+  const navigate = useNavigate() 
+  const handleClick = useCallback( () => { navigate(id)}, [id] )
+
+  return (
+    <Container>
+      <Card style={{ width: '14rem' }} border='dark'>
+        <Card.Body id='rooms-card' onClick={handleClick}>
```

Comment:
`rooms-card` should be a `className` since it will be used in several `HTML elements` in the same page.